sdk/storage/inc/datalake/datalake_utilities.hpp

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include "common/crypt.hpp"
+
+#include <map>
+#include <string>
+
+namespace Azure { namespace Storage { namespace DataLake {
+
+  inline std::map<std::string, std::string> ParseDataLakePropertiesString(
```

Comment:
We're already in `DataLake` namespace, is it necessary to repeat it in function name?

Reply:
I'll remove DataLake.